### ISSUE TITLE
API to take Screenshot

### ISSUE THREAD

**Issue by nunes-lucas** [wppconnect-team/wppconnect-server#1216]
**Describe the solution you'd like**
A new API endpoint to receive a page screenshot image for a given chat number

**Comment by muneebanwar33** [comment]
Yes That will be much help full as other api also have this like open-wa but not for a specific contact for whole whatsapp

**Comment by alexandreluchini** [comment]
It would be great to have this option in the API

**Comment by icleitoncosta** [comment]
In implementing by https://github.com/wppconnect-team/wppconnect-server/commit/1cab9aa2049ecc0decd6f95b766c75ea7acf25a8
